From 977a435d4ef9cd7c4d71918d7f81cb5562f1bd94 Mon Sep 17 00:00:00 2001
From: Adamantcheese <Adamantcheese@users.noreply.github.com>
Date: Thu, 15 Oct 2020 00:17:46 -0700
Subject: [PATCH] remove leakcanary for CI

---
 Kuroba/app/build.gradle | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Kuroba/app/build.gradle b/Kuroba/app/build.gradle
index ad5cb11a1..f4eecfef1 100644
--- a/Kuroba/app/build.gradle
+++ b/Kuroba/app/build.gradle
@@ -254,8 +254,6 @@ dependencies {
     testImplementation 'com.squareup.okhttp3:mockwebserver:4.8.0'
     testImplementation 'org.mockito:mockito-core:3.4.6'
     testImplementation 'com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0'
-
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:2.4'
 }
 
 // Download the current archives.json
-- 
2.20.1.windows.1

